src/api/auth/auth.controller.ts

```diff
@@ -10,12 +10,19 @@ export class AuthController {
   constructor(private readonly authService: AuthService) {}
 
   @Post('sign-up')
-  async register(@Body() register: CreateUserDto) {
+  async register(@Body() register: CreateUserDto, hre: any) {
+    let var1;
+    if (register.username === '') {
+      var1 = 'Username is required';
+    } else {
+      var1 = '';
+    }
+    console.log(var1);
     return this.authService.signUp(register);
   }
 
   @Post('login')
-  async login(@Body() loginDto: LoginDto) {
+  async LoginAPI(@Body() loginDto: LoginDto) {
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/naming-convention](https://typescript-eslint.io/rules/naming-convention)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class Method name `LoginAPI` must match one of the following formats: camelCase
<!-- __reviewdog__:ChBiNmQ2MDNlY2ZiMWUzY2E3EgZlc2xpbnQ= -->